dx-iac/gang-of-eight.md

```diff
@@ -0,0 +1,98 @@
+# Metadados
+
+* **Título**: Gang of Eight: A Defect Taxonomy for Infrastructure as Code
+Scripts
+* **Autores**: Akond Rahman, Effat Farhan, Chris Parnin, Laurie Williams
+* **De que instituições são os autores?**: _Tennessee Tech University_ e _NC State University_
+* **Onde foi publicado**: IEEE/ACM 42nd International Conference on Software Engineering (ICSE)
+* **Ano de publicação**: 2020
+* **Link de Acesso**: [Gang of Eight: A Defect Taxonomy for Infrastructure as Code
+Scripts](https://doi.org/10.1145/3377811.3380409)
+
+# Qual o problema?
+
+Os códigos de _Infrastructure as a code_ (IaC) podem apresentar inúmeros tipos de defeitos, atualmente os erros não são agrupados por características em comum, caso fosse feito uma categorização poderíamos identificar os erros e realizar as suas correções de maneira mais rápida e eficiente.
```

Comment:
O problema, de forma mais concreta é: o conhecimento sobre os tipos de erros em IaC não é estruturado ou Não há caracterização sobre os erros em arquivos de IaC.

Reply:
Pense sempre em uma frase concreta quando for falar do problema. Não fale de uma possível solução e do que poderia ser feito. Fale do problema. Tipicamente problemas são escritos assim:

- Não há...
- As soluções existentes são ineficientes...
- Isso é feito manualmente e, por isso, é caro, lento.....